server/model/spans.go

```diff
@@ -9,6 +9,18 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
+type TracetestMetadataField string
```

Comment:
do we really need this type? it seems that all the references to values for this convert it to string, and no other operation is done. Given that this is just a constant to avoid repetition, and it's very unlikely that we'd want to attach methods to it, does it make sense to keep it as a simple "string"?

Reply:
Done 😄